transport.go

```diff
@@ -15,28 +19,28 @@ type Transport struct {
 	// Base is the base RoundTripper used to make HTTP requests. If nil, then
 	// http.DefaultTransport is used
 	Base http.RoundTripper
-	// source supplies the token to use when signing a request
-	source TokenSource
-	// auther adds OAuth1 Authorization headers to requests
-	auther *auther
+	// Source supplies the token to use when signing a request
+	Source TokenSource
+	// Auther adds OAuth1 Authorization headers to requests
+	Auther Auther
```

Comment:
provide init function might be better than exposing private field ?

Reply:
thanks fixed in 4746ab9212638bb7d5ce985df5896bef3054a103